src/python/grpcio_tests/tests_aio/unit/channel_test.py

```diff
@@ -12,23 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import asyncio
 import logging
 import unittest
 
 import grpc
 
 from grpc.experimental import aio
-from tests_aio.unit import test_base
 from src.proto.grpc.testing import messages_pb2
 from tests.unit.framework.common import test_constants
+from tests_aio.unit._test_server import start_test_server
+from tests_aio.unit._test_base import AioTestBase
 
 
-class TestChannel(test_base.AioTestBase):
+class TestChannel(AioTestBase):
 
     def test_async_context(self):
 
         async def coro():
-            async with aio.insecure_channel(self.server_target) as channel:
+            server_target, unused_server = await start_test_server()
```

Comment:
what's the downside effect of not closing the server? having started as many servers as many test cases we have? if so, we should address this once we have the `stop` method available for the server.

Reply:
It leaks C-Core server object and occupied the port forever. Yes, we should `stop` the server after use, even could using context manager to do it.

---

Shutdown process PR is here: https://github.com/grpc/grpc/pull/20805
C-Core objects are designed in a way that if we want to support graceful shutdown, we need some toil in wrapper layer.